examples/asa/scripts/3-contract-owned-asa.js

```diff
@@ -0,0 +1,33 @@
+const { executeTransaction } = require('@algo-builder/algob');
+const { mkParam } = require('./transfer/common');
+
+/**
+* only owner can call - update smart contract to set a new owner
+* only owner can transfer ASA out from the smart contract and maximum payment is 100 ASA.
+
+let's put this example in `examples/asa` and:
+* update README
+* update documentation
+* if needed - let's create a highl level function for this (similar to `opIn*` functions).
+ * @param {*} runtimeEnv
+ * @param {*} deployer
```

Comment:
Please update the comment. There shouldn't be "let's put this example in ..."

Reply:
Yes, removing